test cases/common/220 fs module/meson.build

```diff
@@ -39,7 +39,9 @@ assert(fs.expanduser('~/foo').endswith('foo'), 'expanduser with tail failed')
 # -- as_posix
 assert(fs.as_posix('/') == '/', 'as_posix idempotent')
 assert(fs.as_posix('\\') == '/', 'as_posix simple')
-assert(fs.as_posix('\\\\') == '/', 'as_posix simple')
+# Python 3.12 changed how these paths are handled, so deal with both.
+drivepath = fs.as_posix('\\\\')
+assert(drivepath == '/' or drivepath == '//', 'as_posix simple')
```

Comment:
maybe we should change the text of the message to make it different from the message of the previous assertion.

Reply:
If this needs further changes then I'll change it. If this goes green I'll release as-is because it is already overdue. In that case patches welcome.